scripts/index.js

```diff
@@ -80,6 +80,9 @@ function fire() {
 		case 'movies':
 			sessionData = movieData.slice(0);
 			break;
+        case 'hindimovies':
```

Comment:
Indentation @an-mittal, what

Reply:
its correct. Github is showing it wrongly

Reply:
@captn3m0 I'll add .editorconfig?

Reply:
Yes please.